src/distributions/mod.rs

```diff
@@ -158,7 +158,7 @@ use crate::Rng;
 /// *   Tuples (up to 12 elements): each element is generated sequentially.
 /// *   Arrays (up to 32 elements): each element is generated sequentially;
 ///     see also [`Rng::fill`] which supports arbitrary array length for integer
-///     types and tends to be faster for `u32` and smaller types.
+///     and float types and tends to be faster for `u32` and smaller types.
```

Comment:
This is a little ambiguous: where exactly is `fill` faster and where does it produce different results? This doc was written before `fill` supported floats (for `AsBytesMut` trait IIRC).

Reply:
Fair enough, this should be verified in benchmarks as well.

Reply:
@vks we have benches:
```
$ cargo +nightly bench --bench misc gen_1kb

running 8 tests
test gen_1kb_u16_fill        ... bench:         137 ns/iter (+/- 0) = 7474 MB/s
test gen_1kb_u16_gen_array   ... bench:         445 ns/iter (+/- 2) = 2301 MB/s
test gen_1kb_u16_iter_repeat ... bench:         485 ns/iter (+/- 0) = 2111 MB/s
test gen_1kb_u16_sample_iter ... bench:         660 ns/iter (+/- 2) = 1551 MB/s
test gen_1kb_u64_fill        ... bench:         128 ns/iter (+/- 2) = 8000 MB/s
test gen_1kb_u64_gen_array   ... bench:         146 ns/iter (+/- 0) = 7013 MB/s
test gen_1kb_u64_iter_repeat ... bench:         123 ns/iter (+/- 0) = 8325 MB/s
test gen_1kb_u64_sample_iter ... bench:         150 ns/iter (+/- 1) = 6826 MB/s

test result: ok. 0 passed; 0 failed; 0 ignored; 8 measured; 6 filtered out; finished in 8.52s
```